faster_sam/cloudformation.py

```diff
@@ -107,6 +225,22 @@ def load(self, template: Optional[str] = None) -> Dict[str, Any]:
             return yaml.load(fp, CFLoader)
 
     def find_nodes(self, tree: Dict[str, Any], node_type: NodeType) -> Dict[str, Any]:
+        """
+        Finds nodes of a specific type in the CloudFormation template.
+
+        Parameters
+        ----------
+        tree : Dict[str, Any]
+            Dictionary representing a tree of the CloudFormation template.
```

Comment:
Aqui eu trocaria a palavra `tree` da descrição do parâmetro por `subset`.

_Dictionary representing a `subset` of the CloudFormation template._